components/prism-bbj.js

```diff
@@ -0,0 +1,21 @@
+(function (Prism) {
+	Prism.languages.bbj = {
+		'comment': [
+			{
+				pattern: /(^|[^\\:])rem\s+.*/i,
+				lookbehind: true,
+				greedy: true
+			}
+		],
```

Comment:
Since there is only one pattern, you don't need the array.
```suggestion
		'comment': {
			pattern: /(^|[^\\:])rem\s+.*/i,
			lookbehind: true,
			greedy: true
		},
```